rest-dashboard.go

```diff
@@ -116,6 +116,42 @@ func (r *Client) GetDashboardBySlug(ctx context.Context, slug string) (Board, Bo
 	return r.getDashboard(ctx, path)
 }
 
+// DashboardVersion represents a response from /api/dashboards/id/:dashboardId/versions API
+type DashboardVersion struct {
+	ID            int       `json:"id"`
+	DashboardID   int       `json:"dashboardId"`
+	ParentVersion int       `json:"parentVersion"`
+	RestoredFrom  int       `json:"restoredFrom"`
+	Version       int       `json:"version"`
+	Created       time.Time `json:"created"`
+	CreatedBy     string    `json:"createdBy"`
+	Message       string    `json:"message"`
+}
+
+// GetDashboardVersionsByID reflects /api/dashboards/id/:dashboardId/versions API call
+func (r *Client) GetDashboardVersionsByID(ctx context.Context, id uint, start, limit int) ([]DashboardVersion, error) {
```

Comment:
Could you please add an integration test for this like so https://github.com/grafana-tools/sdk/blob/master/rest-alertnotification_integration_test.go? Also, please use a pattern like this https://github.com/grafana-tools/sdk/blob/2de27e0f25774ce8a86e5e5f2da51d808b72632b/rest-dashboard.go#L396 because it allows for easy extension in the future.

Reply:
:ok_hand: